tests/e2e/default/analytics/analytics.wdio-spec.js

```diff
@@ -105,4 +105,18 @@ describe('Targets', () => {
       'Targets are disabled for admin users. If you need to see targets, login as a normal user.'
     );
   });
+  
+  it('Should show error message for bad config', async () => {
+    const settings = await compileTargets('targets-error-config.js');
+    await updateSettings(settings);
+    await analyticsPage.goToTargets();
+
+    const { errorMessage, url, username, errorStack } = await analyticsPage.getErrorLog();
+
+    expect(username).to.equal(chw.username);
+    expect(url).to.equal('localhost');
+    expect(errorMessage).to.equal('Error fetching targets');
+    expect(await (await errorStack.isDisplayed())).to.be.true;
+
```

Comment:
Minor but can we assert:
```suggestion
    expect(await (await errorStack.isDisplayed())).to.be.true;
    expect(await (await errorStack.getText())).to
      .include('TypeError: Cannot read properties of undefined (reading \'muted\')');
```